### PR TITLE
tyr: keeping trace of ed's job duration

### PR DESCRIPTION
**Pull request by kinnou02**
the table metric store the duration of all job, this way we will be able
to see change in performance on data integration

### CODE REVIEW

File: source/tyr/tyr/binarisation.py [hunk 1]
```diff
@@ -129,7 +138,10 @@ def fusio2ed(self, instance_config, filename, job_id):
         connection_string = make_connection_string(instance_config)
         params.append("--connection-string")
         params.append(connection_string)
+        begin = datetime.datetime.utcnow()
```
Comment (antoine-de) [inline source/tyr/tyr/binarisation.py]: maybe you can use a contextmanager here

**Comment by kinnou02** [comment]
done

**Comment by Hoshiyo** [comment]
Jenkins, test this please

**Comment by antoine-de** [comment]
Jenkins, test this please

**Pull request merged**